#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_ansi_c.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## DP: Make sure the code compiles when using -ansi.  Renames non-ANSI
## DP: C 'inline' to '__inline'.

@DPATCH@
Index: dmidecode/biosdecode.c
===================================================================
RCS file: /home/pere/src/cvsroot/src/dmidecode/biosdecode.c,v
retrieving revision 1.6
diff -u -3 -p -u -r1.6 biosdecode.c
--- dmidecode/biosdecode.c	1 Mar 2005 22:04:56 -0000	1.6
+++ dmidecode/biosdecode.c	1 Mar 2005 22:09:23 -0000
@@ -502,7 +502,7 @@ static struct bios_entry bios_entries[]=
 };
 
 /* Believe it or not, this is significantly faster than memcmp and strncmp */
-static inline int anchor_match(const struct bios_entry *entry, const char *p)
+static __inline int anchor_match(const struct bios_entry *entry, const char *p)
 {
 	size_t i;
 	
--- dmidecode-2.9.orig/types.h
+++ dmidecode-2.9/types.h
@@ -32,7 +32,7 @@
 #endif
 
 #ifdef ALIGNMENT_WORKAROUND
-static inline u64 U64(u32 low, u32 high)
+static __inline u64 U64(u32 low, u32 high)
 {
 	u64 self;
 	
